### ISSUE TITLE
Randomizing list bug

### ISSUE THREAD

**Issue by TimeTravelPenguin** [aGlitch/Mario-64-Randomizer#1]
Heyo, I am just reviewing the code, and I found a bug when doing some general clean up. Nothing too complex, but the following line has an issue:

In `btnEnterSeed_Click`:
```randomList = randomList.GetRange(0, Convert.ToInt32(nudStarAmount.Value));```

If `randomList` has less elements than `nudStarAmount.Value`, you will get `System.ArgumentException`.

I am currently looking through the code, and it seems like an easy fix I can push shortly.

**Comment by TimeTravelPenguin** [comment]
I have forked my own version of this code and applied a fix. I have made a pull request which fixes the issue. If you would like, I would be happy to help with this project. Let me know, and I can inform you on how to contact me.

**Comment by TimeTravelPenguin** [comment]
Whoops, I forgot to output what I did in my previous comment!

Simply, you had two lines of code in both the randomize seen and set seed button.
I moved this buggy code into a new method, so it is more simple to alter and fix in possible future incidents.

```
if (randomList.Count > nudStarAmount.Value)
    {
        randomList = randomList.GetRange(0, Convert.ToInt32(nudStarAmount.Value));
    }
```

The code was left unchanged but put into an if statement where, if `randomList` contains more stars than the number of starts the user wants to use in `nudStarAmount`, it will run. Otherwise, it will maintain and display the current list size, as it cannot possibly output more than it holds. Simple fix.

Other than that, I am working on a bunch of cleanups to make the code much cleaner and futureproof to additional features. I am also interested in creating a more interactive interface, but this isn't the place for that discussion.
